tests/nightly/install-snaps/task.yaml

```diff
@@ -62,7 +62,6 @@ environment:
     SNAP/go: go
     SNAP/juju: juju
     SNAP/neutron: neutron
-    SNAP/nova: nova
```

Comment:
Do you have an example error log? Maybe we should pull out nova and neutron into a separate OpenStack oriented test?

Reply:
@bboozzoo https://travis-ci.org/snapcore/spread-cron/builds/613287688#L2576

Reply:
@bboozzoo also the gimp snap started failing on ubuntu-16.04-32, I dont know if users are using it on that architecture but seems to be real dependencies issue, what do you think?

Reply:
I think those are concerning and should be reported to other teams.

Gimp is a real regression that the desktop team AFAIK knows about now.

Reply:
@zyga I already reported gimp to the desktop team, they found the problem is for all the architectures but amd64. That should be automatically fixed once the dependency is uploaded again.